src/components/Display/Display.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { useParams } from 'react-router-dom';
+
+export default function Display() {
+  const { r, g, b } = useParams();
+  return (
+    <div style={{ backgroundColor: `rgb(${r}, ${g}, ${b})` }}>
+      <p style={{ height: 1000 }}> rgb({r}, {g}, {b})</p>
```

Comment:
I usually recommend that you stay away from inline styling.  this is fine since it is one thing but if your not using a variable to set styling try and keep it in the css file.